services/dex-k8s-authenticator/1.3.3/dex-k8s-authenticator.yaml

```
@@ -21,7 +21,7 @@ spec:
         kind: HelmRepository
         name: mesosphere.github.io-charts-staging
         namespace: kommander-flux
-      version: 1.3.2
+      version: 1.3.3
```

```suggestion
      version: 1.3.2
```

I think this is just a config change, doesn't look like we published a new helm chart version?